app/lib/error.js

```diff
@@ -49,11 +49,13 @@ const error_ = {
   },
 
   // Log and report formatted errors to the server, without throwing
-  report (message, context = {}) {
+  report (message, context = {}, statusCode = 599) {
     context = deepClone(context)
     context.location = location.href
-    // Non-standard convention: 599 = client implementation error
-    const err = formatError(message, 599, context)
+    // Non-standard convention
+    // 599 = client implementation error
+    // 598 = user abuse
```

Comment:
```suggestion
    // 598 = suspected user abuse or spam account
```
Non-standard requires being having a precise documentation imo

Reply:
done in 0888cd876